libatalk/bstring/bstrlib.c

```diff
@@ -2737,7 +2737,7 @@ struct genBstrList g;
 #define START_VSNBUFF (256)
 #else
 
-#ifdef __GNUC__
+#if defined(__GNUC__) && !defined(__clang__)
```

Comment:
I think this inline prototype is bitrotted. Just now I put up a PR to remove it. https://github.com/Netatalk/netatalk/pull/316

Reply:
Great, thanks for this. This particular patch has been discarded from my PR branch...